docs/sources/datasources/aws-cloudwatch/_index.md

```diff
@@ -79,6 +79,12 @@ Here is a minimal policy example:
 
 Grafana is not able to load custom namespaces through the GetMetricData API. If you still want your custom metrics to show up in the fields in the query editor, you can specify the names of the namespaces containing the custom metrics in the _Namespaces of Custom Metrics_ field. The field accepts a multiple namespaces, separated by a comma.
 
+#### CloudWatch Logs
+
+##### Timeout
```

Comment:
@sunker changed but does it make sense to nest this so deep? I mean other sections does not seem to go field by field as the config is more complex but here it seems appropriate to do description per field (even though there is only one). Maybe using a list? But that will also be only section using a list it seems.

Reply:
No I agree it's too deep. I'm hoping to be able to refactor this a lot in the near future. Currently, we're referencing another document (AWS authentication) for the shared parts, but rather than referencing this document, I'd like to use it as a template. When doing so, we can revisit the heading. But for now, I think it's best to just use the same heading pattern as everything else. 

What you could do though it to remove the CloudWatch Logs header and make Timeout an H4. The timeout text makes it obvious that this is only applicable for CloudWatch logs anyway. 

Sorry if I'm sending mixed messages when reviewing this!